inlong-manager/manager-pojo/src/main/java/org/apache/inlong/manager/pojo/user/UserRoleCode.java

```diff
@@ -25,4 +25,8 @@ public class UserRoleCode {
     public static final String ADMIN = "ADMIN";
     public static final String OPERATOR = "OPERATE";
 
+    public static final String INLONG_ADMIN = "INLONG_ADMIN";
+
+    public static final String INLONG_OPERATOR = "INLONG_OPERATE";
```

Comment:
What is the difference between `ADMIN` and `INLONG_ADMIN`?

What's more, the `"OPERATOR"` is more clear than the `"OPERATE"`.

Reply:
**_ADMIN_** only has the permission of its own tenant, while **_INLONG_ADMIN_** has permission of all tenant. 
The permission check will be done at those **servlet filters**